app/mailers/user_mailer.rb

```diff
@@ -27,37 +29,36 @@ def member_invitation(invite)
     mail(to: @invite.email, subject: 'Epic Logger Invite Users')
   end
 
-  def error_occurred(website_id, message_id)
+  def error_occurred(issue)
     counter = 0
-    @message = Message.find(message_id)
-    @website = Website.find(website_id)
+    @issue = issue
+    @website = issue.website
     @website.grouped_issues.each do |group|
       counter += Issue.where('group_id = ? AND created_at > ?', group.id, Time.now - 1.hour).count
     end
     if counter >= 10
-      more_than_10_errors(website_id, message_id)
+      more_than_10_errors(@website.id, message_id)
     else
-      mail(subject: "Epic Logger Realtime Error",bcc: mail_to(website_id))
+      mail(subject: "Epic Logger Realtime Error",bcc: mail_to(@website))
```

Comment:
It's safe to do a ` mail to: owners.join(','),` here instead of doing bcc that's going to show to all owners who got the email. 

If you use to with comma then each will receive an email for themselves without the cc attribute for reply to.